.flowconfig

```diff
@@ -0,0 +1,81 @@
+[ignore]
+; We fork some components by platform
+.*/node_modules/.*/*[.]android.js
+
+; Ignore "BUCK" generated dirs
+<PROJECT_ROOT>/\.buckd/
+
+; Ignore unexpected extra "@providesModule"
+.*/node_modules/.*/node_modules/fbjs/.*
+
+; Ignore duplicate module providers
+; For RN Apps installed via npm, "Libraries" folder is inside
+; "node_modules/react-native" but in the source repo it is in the root
+.*/Libraries/react-native/React.js
+
+; Ignore polyfills
+.*/Libraries/polyfills/.*
+
+; Ignore metro
+.*/node_modules/metro/.*
+
+; My custom ignores
+<PROJECT_ROOT>/\.git/
+<PROJECT_ROOT>/node_modules/reqwest/.+
+<PROJECT_ROOT>/node_modules/fbemitter/.+
+<PROJECT_ROOT>/node_modules/xdl/build/detach/.+
+<PROJECT_ROOT>/node_modules/react-relay/lib/.+
+<PROJECT_ROOT>/node_modules/relay-compiler/lib/.+
+<PROJECT_ROOT>/node_modules/relay-runtime/lib/.+
+<PROJECT_ROOT>/node_modules/react-native-web/.+
+
+[include]
+
+[untyped]
+<PROJECT_ROOT>/node_modules/react-native/Libraries/.+
+<PROJECT_ROOT>/node_modules/expo/.+
+<PROJECT_ROOT>/node_modules/iterall/.+
+<PROJECT_ROOT>/node_modules/graphql/.+
+<PROJECT_ROOT>/node_modules/express-graphql/.+
+
+[libs]
+node_modules/react-native/Libraries/react-native/react-native-interface.js
+node_modules/react-native/flow/
+node_modules/react-native/flow-github/
+flow-typed/
+
+[options]
+emoji=true
+
+module.system=haste
+
+munge_underscores=true
+
+module.name_mapper='^[./a-zA-Z0-9$_-]+\.\(bmp\|gif\|jpg\|jpeg\|png\|psd\|svg\|webp\|m4v\|mov\|mp4\|mpeg\|mpg\|webm\|aac\|aiff\|caf\|m4a\|mp3\|wav\|html\|pdf\)$' -> 'RelativeImageStub'
+
+module.file_ext=.js
+module.file_ext=.android.js
+module.file_ext=.ios.js
+module.file_ext=.web.js
+module.file_ext=.jsx
+;module.file_ext=.json
+module.file_ext=.native.js
+
+; This option lets you alias 'any' with a given string - useful for explaining why you’re using 'any'
+; Please, do not use "fix me" suppress types and comments
+suppress_type=$FlowFixMe
+suppress_type=$FlowFixMeProps
+suppress_type=$FlowFixMeState
+
+suppress_comment=\\(.\\|\n\\)*\\$FlowFixMe\\($\\|[^(]\\|(\\(<VERSION>\\)? *\\(site=[a-z,_]*react_native[a-z,_]*\\)?)\\)
+suppress_comment=\\(.\\|\n\\)*\\$FlowIssue: https://github.com/facebook/flow/issues/[0-9]+
+suppress_comment=\\(.\\|\n\\)*\\$FlowExpectedError: .+
+
+[lints]
+all=error
+sketchy-null=off
+untyped-import=off
+; unclear types are used by Relay Compiler
+unclear-type=off
+; untyped imports are used by Relay Compiler
+untyped-type-import=off
```

Comment:
could you also add 
`esproposal.optional_chaining=enable`
`esproposal.nullish_coalescing=enable`
`unnecessary-optional-chain=error`

see [mobile-flowconfig](https://github.com/kiwicom/mobile/blob/master/.flowconfig#L55)

Reply:
Sure